hexa/pipelines/utils.py

```diff
@@ -0,0 +1,32 @@
+from django.utils.translation import gettext_lazy
+
+from config import settings
+from hexa.core.utils import send_mail
+
+from .models import PipelineRun, PipelineRunTrigger
+
+
+def mail_run_recipients(run: PipelineRun):
+    recipient_list = []
+    if run.trigger_mode == PipelineRunTrigger.MANUAL:
+        recipient_list = [run.user.email]
+    else:
+        recipient_list = [
+            recipient.email for recipient in run.pipeline.recipients.all()
+        ]
+
+    workspace_slug = run.pipeline.workspace.slug
+    send_mail(
+        title=gettext_lazy(f"Run report of {run.pipeline.code}({run.state.name})"),
```

Comment:
Don't forget to indicate the status (failure or success) in the report as well

Reply:
Copy that.